### PR TITLE
[Home] Include all followed genes and fix order.

### LINKED ISSUE #321: Include all categories that a user follows in the home feed
Right now we only pull in one category that a user follows, we should pull in all of them.

### PR DESCRIPTION
**Pull request by alloy**
Fixes #321 & #314.

### CODE REVIEW

File: lib/relay/config.js [hunk 1]
```diff
@@ -10,7 +10,6 @@ if (Emission.useStagingEnvironment) {
   metaphysicsURL = 'https://metaphysics-staging.artsy.net'
 } else {
   metaphysicsURL = 'https://metaphysics-production.artsy.net'
-  // metaphysicsURL = 'http://localhost:5100'
```
Comment (orta) [inline lib/relay/config.js]: ah yeah, I made a mental todo about that - but never got around to it

Comment (alloy) [inline lib/relay/config.js]: I got your back, buddy 🍷

**Comment by sarahscott** [comment]
✨

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/016bf82eb81f226b674768cc0ce38ff4574526fd/lib/containers/home.js">lib/containers/home.js</a> &amp; <a href="https://github.com/artsy/emission/blob/016bf82eb81f226b674768cc0ce38ff4574526fd/lib/relay/config.js">lib/relay/config.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by alloy** [comment]
Mergin’!

**Pull request merged**